common/app/views/fragments/contentMeta.scala.html

```diff
@@ -56,7 +56,7 @@
     @if(item.tags.contributors.length == 1 && item.tags.contributors.headOption.exists(_.twitterHandle.nonEmpty)) { content__meta-container--twitter}
     @if(item.tags.contributors.length == 1 && item.tags.contributors.headOption.exists(_.emailAddress.nonEmpty)) { content__meta-container--email}">
 
-    @if(showBadge) {
+    @if(showBadge && (NewCommercialContent.isSwitchedOff || item.tags.isVideo || item.tags.isAudio)) {
```

Comment:
that I understood, but that showBadge isn't false in Video and Audio pages is kind of baffling.